tests/nopytest_test_coadds.py

```diff
@@ -218,6 +170,85 @@ def runForcedPhotCcdTask(butler):
+        # There's no noise in these images, so we set a direct-value threshold,
+        # and the background weighting (when using Approximate) to False
+
+        detectConfig = DetectCoaddSourcesTask.ConfigClass()
+        # Images have no noise, so we can't use the default DynamicDetectionTask
+        detectConfig.detection.retarget(lsst.meas.algorithms.SourceDetectionTask)
+        detectConfig.detection.thresholdType = "value"
+        detectConfig.detection.thresholdValue = 0.01
+        detectConfig.detection.background.weighting = False
+        detectTask = DetectCoaddSourcesTask(config=detectConfig)
+
+        butler = lsst.pipe.tasks.mocks.makeDataRepo(DATAREPO_ROOT)
+
+        mocksTask.buildAllInputs(butler)
+
+        addMaskPlanes(butler)
+        mocksTask.buildCoadd(butler)
+        mocksTask.buildMockCoadd(butler)
+        detectTask.writeSchemas(butler)
+        # Now run the seperate multiband tasks on the Coadd to make the reference
+        # catalog for the forced photometry tests.
+        runTaskOnPatches(butler, detectTask, mocksTask)
+
+        mergeDetConfig = MergeDetectionsTask.ConfigClass()
+        mergeDetConfig.priorityList = ['r', ]
+        mergeDetTask = MergeDetectionsTask(config=mergeDetConfig, butler=butler)
+        mergeDetTask.writeSchemas(butler)
+        runTaskOnPatchList(butler, mergeDetTask, mocksTask)
+
+        deblendSourcesConfig = DeblendCoaddSourcesTask.ConfigClass()
+        deblendSourcesTask = DeblendCoaddSourcesTask(config=deblendSourcesConfig, butler=butler)
+        deblendSourcesTask.writeSchemas(butler)
+        runTaskOnPatchList(butler, deblendSourcesTask, mocksTask)
+
+        measMergedConfig = MeasureMergedCoaddSourcesTask.ConfigClass()
+        measMergedConfig.measurement.slots.shape = "base_SdssShape"
+        measMergedConfig.measurement.plugins['base_PixelFlags'].masksFpAnywhere = []
+        measMergedConfig.propagateFlags.flags = {}  # Disable flag propagation: no flags to propagate
+        measMergedConfig.doMatchSources = False  # We don't have a reference catalog available
+        measMergedTask = MeasureMergedCoaddSourcesTask(config=measMergedConfig, butler=butler)
+        measMergedTask.writeSchemas(butler)
+        runTaskOnPatches(butler, measMergedTask, mocksTask)
+
+        mergeMeasConfig = MergeMeasurementsTask.ConfigClass()
+        mergeMeasConfig.priorityList = ['r', ]
+        mergeMeasTask = MergeMeasurementsTask(config=mergeMeasConfig, butler=butler)
+        mergeMeasTask.writeSchemas(butler)
+        runTaskOnPatchList(butler, mergeMeasTask, mocksTask)
+
+        runForcedPhotCoaddTask(butler, mocksTask)
+        runForcedPhotCcdTask(butler)
+
+    @classmethod
+    def tearDownClass(cls):
+        """Removes test data if tests ran okay"""
+        if os.path.exists(DATAREPO_ROOT):
+            if not cls.failed:
+                print(f"Deleting temporary data repository {DATAREPO_ROOT}")
+                # Errors can be ignored at this point
```

Comment:
I'm not sure the comment really adds much... presumably, if the errors couldn't be ignored, the code wouldn't say `ignore_errors=True`. :-) I'd suggest either removing the comment, or using it to explain _why_ it's appropriate to ignore errors.